generators/java/sdk/src/main/java/com/fern/java/client/generators/RequestOptionsGenerator.java

```diff
@@ -51,6 +46,11 @@ public final class RequestOptionsGenerator extends AbstractFileGenerator {
                     Modifier.FINAL)
             .build();
 
+    private static final FieldSpec.Builder TIMEOUT_FIELD_BUILDER = FieldSpec.builder(
+                    ParameterizedTypeName.get(ClassName.get(Optional.class), TypeName.get(Integer.class)),
+                    "timeout",
+                    Modifier.PRIVATE);
```

Comment:
in java i think its common to do `timeout(10, TimeUnit.Seconds)` -- thats what okhttp does

Reply:
Will do! I was mirroring some of the others which is just in seconds